deploy/global.yaml

```diff
@@ -91,6 +91,9 @@ components:
     resources:
       requests:
         cpu: 50m
+        memory: 400Mi
+      limits:
+        memory: 400Mi
```

Comment:
We're again getting OOMs in `frontend` service, so bumping these.

Reply:
nextjs 😭